client/components/sites-dropdown/test/index.js

```diff
@@ -23,104 +24,130 @@ describe( 'index', function() {
 		mockery.registerSubstitute( 'matches-selector', 'component-matches-selector' );
 	} );
 
-	let SitesDropdown;
+	describe( 'Connected(SitesDropdown)', function() {
+		let ConnectedSitesDropdown;
 
-	before( function() {
-		SitesDropdown = require( '../index.jsx' );
-	} );
-
-	describe( 'component rendering', function() {
-		it( 'should render a dropdown component initially closed', function() {
-			const sitesDropdown = shallow( <SitesDropdown /> );
-			expect( sitesDropdown.hasClass( 'sites-dropdown' ) ).to.be.true;
-			expect( sitesDropdown.hasClass( 'is-open' ) ).to.be.false;
+		before( function() {
+			ConnectedSitesDropdown = require( '../index.jsx' ).default;
```

Comment:
Interesting case here, another option would be to import the unconnected class, and pass through the sites and selected siteId data. @gziolo do you have any opinions here?